inc/commoninjectionlib.class.php

```diff
@@ -42,7 +42,7 @@ class PluginDatainjectionCommonInjectionLib
     private $mandatory_fields = [];
 
     //List of fields which can agregate more than one value (type multiline_text)
-    private $severalvalues_fields = [];
+    //private $severalvalues_fields = [];
```

Comment:
why ? unused ?

Reply:
Unused

Reply:
In this case, we don't need to keep the line as a comment ?